sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,210 @@
+
+            SapVirtualInstanceData SviCreatePayload = this.GetSingleServerPayloadToPut(rg.Data.Name, false);
+
+            // Create SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: SviName,
+                    data: SviCreatePayload);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            SapVirtualInstanceData SviInstallPayload = this.GetSingleServerPayloadToPut(rg.Data.Name, true);
+
+            // Install SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: SviName,
+                    data: SviInstallPayload);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            var sviObject1 = await rg.GetSapVirtualInstanceAsync(SviName);
+
+            // Delete RG
+            await rg.DeleteAsync(WaitUntil.Completed);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVIStartStopOperations()
+        {
+            string SviName = "C11";
+
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            var lro = await subscription.GetResourceGroups().GetAsync("E2E-SVI-DevBox-27Jun-avzone-SLES-SAP-12-sp4-gen2");
+            ResourceGroupResource rg = lro.Value;
+
+            // Stop operation
+            var sviObject1 = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            await sviObject1.Value.StopAsync(WaitUntil.Completed);
+
+            var sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Offline);
+
+            // Start operation
+            await sviObject1.Value.StartAsync(WaitUntil.Completed);
+            sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Running);
+        }
+
+        /// <summary>
+        /// Gets an object of SAP Workload Resource for Single Server that should be used for PUT call.
+        /// </summary>
+        /// <param name="infraRgName"> Contains the infra RG
+        /// used as app resource group. </param>
+        /// <returns>PHP Workload Resource.</returns>
+        private SapVirtualInstanceData GetSingleServerPayloadToPut(string infraRgName, bool isInstall)
+        {
+            var testSapWorkloadJson = File.ReadAllText(Path.Combine(Directory.GetCurrentDirectory(), @"tests\SingleServerInstall.json"));
```

Comment:
You can see the CI failed because the read file issue, this isn't the best practice to read the file. I think you should first add below code to your test csproj:
```xml
  <ItemGroup>
    <None Update="SingleServerInstall.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```
And then read the file here use:
```csharp
Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
    "SingleServerInstall.json");
```
Please have a try to see if it works, thanks.

Reply:
This worked, thank you.